guides/common/modules/con_host-group-structures.adoc

```diff
@@ -5,20 +5,28 @@ The fact that host groups can be nested to inherit parameters from each other al
 A well planned host group structure can help to simplify the maintenance of host settings.
 This section outlines four approaches to organizing host groups.
 
-[[figu-Life_Cycle_Environment_Based_Structure]]
 .Host group structuring examples
-
-image::host-group-structures-satellite.png[Host group structuring examples]
+ifdef::foreman-el,foreman-deb,katello[]
+image::common/host-group-structures.svg[Host group structuring examples]
+endif::[]
+ifdef::orcharhino[]
+TODO: Add image
+endif::[]
+ifdef::satellite[]
+image::common/host-group-structures-satellite.png[Host group structuring examples]
+endif::[]
 
 Flat structure::
 The advantage of a flat structure is limited complexity, as inheritance is avoided.
 In a deployment with few host types, this scenario is the best option.
 However, without inheritance there is a risk of high duplication of settings between host groups.
 
+ifdef::katello,orcharhino,satellite[]
 Lifecycle environment based structure::
 In this hierarchy, the first host group level is reserved for parameters specific to a lifecycle environment.
 The second level contains operating system related definitions, and the third level contains application specific settings.
-Such structure is useful in scenarios where responsibilities are divided among lifecycle environments (for example, a dedicated owner for the *Development*, *QA*, and *Production* lifecycle stages).
+Such structure is useful in scenarios where responsibilities are divided among lifecycle environments, for example, a dedicated owner for the *Development*, *QA*, and *Production* lifecycle stages.
```

Comment:
Is it just me or does this also look strange without the parentheses? Written this way, it looks like you're listing examples of lifecycle environments (nonsense, I know, but that's what it looks like when I read it as one sentence without the extra parenthesis separator). Would it make sense to split this into two sentences as well?

Reply:
Reworded. What do you think now?